examples/fargate/README.md

```diff
@@ -0,0 +1,58 @@
+# ECS Clusters w/ Fargate
+
+Configuration in this directory creates:
+
+- ECS cluster using Fargate (on-demand and spot) capacity provider
```

Comment:
Once the service sub-module is added, its usage will be added here along with example usage of using ALB and NLB to route traffic to tasks.